docs/PinnedImageSets.md

```diff
@@ -0,0 +1,35 @@
+# Updating a cluster in a disconnected environment without a local registry
+
+You can use `PinnedImageSet` custom resources (CRs) to pin and pre-load release images to a defined machine config pool, which enables you to update an OpenShift Container Platform (OCP) cluster in a disconnected environment without needing an image registry in the environment.
+This can be useful for clusters that were installed using the [OpenShift-based Appliance builder](https://access.redhat.com/articles/7065136), which might have been deployed in disconnected environments where building a local image registry is not practical.
+
+**Note**: To update a cluster in a connected environment, or in a disconnected environment with a registry, see the product documentation for [updating a cluster](https://docs.openshift.com/container-platform/4.16/updating/understanding_updates/intro-to-updates.html).
+
+## Prerequisites
+
+* You have access to your cluster with administrator privileges.
+
+## Procedure
+
+1. Go to the [OpenShift Container Platform release repository](https://quay.io/repository/openshift-release-dev/ocp-release) and download the release images for your target update version onto a removable media, such as a USB drive.
+**Tip**: You can find update recommendations and update paths on the [Red Hat OpenShift Container Platform Update Graph Application](https://access.redhat.com/labs/ocpupgradegraph/update_path).
+
+2. Mount the removable media onto your cluster.
```

Comment:
I am also unsure whether this is the right action or all that needs to be done. Does the user need to move the images into a specific place on the cluster? Do they just need to note the filepath to the USB/device and use that path in the `PinnedImageSet` CR?